monitor/monitoring-dam-generator-core/src/main/java/eu/seaclouds/monitor/monitoringdamgenerator/dcgenerators/MODACloudsDcGenerator.java

```diff
@@ -17,10 +15,10 @@
             .getLogger(MODACloudsDcGenerator.class);
 
     private static final String MODACLOUDS_DC_ID = "modacloudsDc";
-    private static final String START_SCRIPT_URL = "http://start.sh/";
+    private static final String START_SCRIPT_URL = "http://installMODACloudsDc.sh/";
```

Comment:
I would upload to Amazon or something like this to have before accepting the PR the resource url ready, but pointing to a github file in a quick PR will be accepted is fine also for me